service/provider/src/main/java/gov/va/vro/service/provider/camel/MasIntegrationRoutes.java

```diff
@@ -178,52 +180,72 @@ private void configureMasProcessing() {
         .to(END_POINT_RFD)
         .otherwise()
         // Off ramp if the Sufficient For Fast Tracking is null
-        .to(ENDPOINT_ACCESS_ERR)
+        .setProperty("offRampError", constant("Sufficiency cannot be determined."))
+        .setProperty("sourceRoute", constant("assessorError"))
+        .log("Assessor Error. Off-ramping claim")
```

Comment:
ENDPOINT_ACCESS_ERR was very close to what I needed from ENDPOINT_OFFRAMP_ERROR and in the interest of not causing too many camel routes and making the code harder, it's been collapsed and what wasn't contained moved here.